CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### 1.5.0
```

Comment:
I wasn't sure about the version number at first, but it looks like there are no public deletions, since `purchases-hybrid-common` masks the changes in `purchases-android`, so I went with 1.5.0